api/base/views.py

```diff
@@ -813,10 +813,10 @@ class BaseNodeLinksDetail(JSONAPIBaseView, generics.RetrieveAPIView):
     def get_queryset(self):
```

Comment:
This function is only called in `ListViews`, not `DetailViews`, which explains why no tests were failing. Removing.